sdk/src/main/java/com/google/cloud/dataflow/sdk/util/BigQueryTableRowIterator.java

```diff
@@ -427,7 +427,8 @@ private TableReference executeQueryAndWaitForCompletion()
       Object... errorArgs) throws IOException, InterruptedException {
     Sleeper sleeper = Sleeper.DEFAULT;
     BackOff backOff =
-        new AttemptBoundedExponentialBackOff(MAX_RETRIES, INITIAL_BACKOFF_TIME.getMillis());
+        FluentBackoff.DEFAULT
```

Comment:
nit: Should `FluentBackoff` be `FluentBackOff` and `backoff` be `backOff` so that capitalization is consistent with `BackOff`?